Wikipedia/Code/WMFLegacyDatabaseHouseKeeper.h

```diff
@@ -1,6 +1,6 @@
 #import <Foundation/Foundation.h>
 
-@interface WMFDatabaseHouseKeeper : NSObject
+@interface WMFLegacyDatabaseHouseKeeper : NSObject
```

Comment:
Yes. Was leaving it around for reference because the code is non-trivial and they're aren't any tests to verify the new behavior. I did hand-verify it though.